Decisions.Equifax/EquifaxSettings.cs

```diff
@@ -18,25 +15,32 @@ namespace Decisions.Equifax
     public class EquifaxSettings : AbstractModuleSettings, IInitializable, IValidationSource
     {
         [ORMField]
-        [PropertyClassification(0, "OAuth Url", "OAuth Settings")]
+        [PropertyClassification(new string[] {"OAuth Settings"} , "OAuth URL",4)]
         public string EquifaxOAuthUrl { get; set; }
 
         [ORMField]
-        [PropertyClassification(1, "API Client ID", "OAuth Settings")]
+        [PropertyClassification(new string[] {"OAuth Settings"} , "API Client ID", 5)]
         public string EquifaxClientId { get; set; }
 
         [ORMField]
-        [PropertyClassification(2, "API Client Secret", "OAuth Settings")]
+        [PropertyClassification(new string[] {"OAuth Settings"} , "API Client Secret", 6)]
         public string EquifaxClientSecret { get; set; }
         
         [ORMField]
-        [PropertyClassification(3, "API Endpoint Url", "Consumer Credit Report")]
+        [PropertyClassification(new string[] {"PreQualification of One"} , "API Endpoint Url", 2)]
```

Comment:
I think for the user facing stuff we can use 'Prequalification'. The camel casing is just for code.